src/main/java/de/retest/web/RecheckSeleniumAdapter.java

```diff
@@ -106,8 +106,10 @@ public RootElement convertToPeers( final Map<String, Map<String, Object>> data,
 				peer = root;
 			} else {
 				peer = new WebElementPeer( defaultValuesProvider, webData, path );
-				final WebElementPeer parent = converted.get( parentPath );
-				assert parent != null : "We sorted the map, parent should already be there.";
+				WebElementPeer parent = converted.get( parentPath );
+				if ( parent == null ) {
+					parent = new WebElementPeer( defaultValuesProvider, webData, parentPath );
```

Comment:
This is wrong: You create the parent with the child data... you need to get the webData for the parent, put the parent into `converted` and add this child to the parent...